README.md

```diff
@@ -0,0 +1,50 @@
+# libholiday
+
+[![Build Status](https://travis-ci.org/mayflower/libholiday.svg?branch=master)](https://travis-ci.org/mayflower/libholiday)
+
+A library to calculate holidays.
+
+Supported countries and states
+
+* America
+* Germany and all constituent states
+* Sweden
+* Netherlands
+
+Usage
+
+Add to your projects ```composer.json```
+
+```json
+{
+    "name": "your/project",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/mayflower/libholiday"
```

Comment:
Do we really want users to use this git repo?

Reply:
You mean like add it to packagist or the like?

Reply:
Isn't this the repo for https://packagist.org/packages/mayflower/holiday?

Reply:
You are completely correct. I did only try mayflower/libholiday, damnit.